Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.6-alpine
+
+RUN adduser -D instadam
+
+WORKDIR /home/instadam
+
+COPY requirements requirements
+RUN \
+ apk add --no-cache postgresql-libs && \
+ apk add --no-cache --virtual .build-deps gcc musl-dev postgresql-dev
+
+RUN pip install --no-cache-dir -r requirements/prod.txt
+RUN pip install gunicorn
```

Comment:
Maybe add this one to the prod.txt?

Reply:
IMO we are using gunicorn as part of our deployment method but not as a requirement for our app. Our users can choose any wsgi server to host their own app and that's why I didn't add it to production requirement.